crates/daphne/src/vdaf/mastic.rs

```diff
@@ -233,19 +202,22 @@ mod test {
     #[test]
     fn roundtrip_count() {
         let mut t = AggregationJobTest::new(
-            &VdafConfig::Mastic {
-                input_size: 4,
+            &VdafConfig::Mastic(MasticConfig {
+                bits: 32,
                 weight_config: MasticWeightConfig::Count,
-            },
+            }),
             HpkeKemId::X25519HkdfSha256,
             DapVersion::Latest,
         );
         let got = t.roundtrip(
             DapAggregationParam::Mastic(
-                Poplar1AggregationParam::try_from_prefixes(vec![
-                    IdpfInput::from_bytes(b"cool"),
-                    IdpfInput::from_bytes(b"trip"),
-                ])
+                MasticAggregationParam::new(
+                    vec![
+                        IdpfInput::from_bytes(b"cool"),
+                        IdpfInput::from_bytes(b"trip"),
+                    ],
+                    true,
```

Comment:
what does this `true` mean?

Reply:
It's a detail of Mastic. It tells the aggreagtors to do perform a specific validity check.

This check should be mandatory, but DAP-13 doesn't enforce it. This is a bug, which we plan to fix in the next version: https://github.com/ietf-wg-ppm/draft-ietf-ppm-dap/pull/649

This is tracked by https://github.com/cloudflare/daphne/issues/770.

Reply:
so the bool parameter should disappear in the future?

Reply:
ah okay. Sure